src/System.Threading.Tasks.Extensions/ref/System.Threading.Tasks.Extensions.csproj

```diff
@@ -16,6 +16,9 @@
   <ItemGroup>
     <Compile Include="System.Threading.Tasks.Extensions.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'net45'">
+   <Reference Include="mscorlib" />
```

Comment:
Why did you need this change?

Reply:
It was from an earlier attempt where I was adding a net45 build, which I didn't end up needing so I'll remove this.